developer_guide.md

```diff
@@ -1,3 +1,60 @@
+
+## Directories Layout
+
+```sh
+$ tree -d -I 'vendor|bin|.git'
+├── build
+│   ├── images
+│   │   └── tf_operator
+│   └── release
+├── cmd
+│   └── tf_operator
+│       └── app
+│           └── options
+├── dashboard
+│   ├── backend
+│   │   ├── client
+│   │   └── handler
+│   └── frontend
+│       ├── public
+│       └── src
+│           └── components
+├── docs
+│   └── diagrams
+├── examples
+│   ├── charts
+│   │   ├── tensorboard
+│   │   └── tf-job-operator-chart
```

Comment:
per conversation tf-job-operator-chart should not be under examples.

Reply:
LGTM.